### PR TITLE
Preset fixes and UI refresh

### PR DESCRIPTION
**Pull request by bgins**
# Description

This PR implements the following features/fixes:

- [x] Remove `selectedPatch` from presets store (we don't select patches here, just share or collect them with checkboxes)
- [x] Improve deduplication and loading of collectable presets
- [x] Check if a preset's `notes` are defined before showing them and truncate overly long notes
- [x] Improve styling of presets tables for dark and light modes
- [x] Add `'lookup' | 'scanning' | 'no-presets'` views to `PresetsCollect` modal
- [x] Move button to collect presets to top right of presets box and label it "Find Presets"
- [x] Add the new Ditto logo

## Link to issue

Please add a link to any relevant issues/tickets

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change that fixes an issue)
- [x] New feature (non-breaking change that adds functionality)

## Screenshots/Screencaps

Light mode:

light

Dark mode:

dark

Modal states:

modal1

modal2

modal3

### CODE REVIEW

**Review by avivash — APPROVED** [review]
Nice! this is all looking great 💯 one idea for the `Find Presets` button could be to show it in the bottom right corner of the table, similar to how the plugin does it 🤷🏼

File: src/components/Header.svelte [hunk 1]
```diff
@@ -2,13 +2,11 @@
...
   import { sessionStore, themeStore } from '../stores'
   import { DEFAULT_THEME_KEY, storeTheme, type ThemeOptions } from '$lib/theme'
-  import AlphaTag from '$components/nav/AlphaTag.svelte'
   import Avatar from '$components/settings/Avatar.svelte'
-  import BrandLogo from '$components/icons/BrandLogo.svelte'
-  import BrandWordmark from '$components/icons/BrandWordmark.svelte'
   import DarkMode from '$components/icons/DarkMode.svelte'
   import Hamburger from '$components/icons/Hamburger.svelte'
   import LightMode from '$components/icons/LightMode.svelte'
+  import Logo from './icons/Logo.svelte'
```
Comment (avivash) [inline src/components/Header.svelte]: maybe import it from `$components/icons/Logo.svelte` so it's consistent with the other imports? not a big deal tho

Comment (bgins) [reply src/components/Header.svelte]: Oh yeah, good call! 💯 Overlooked that when bringing in the new logo.

**Comment by bgins** [comment]
> Nice! this is all looking great 💯 one idea for the `Find Presets` button could be to show it in the bottom right corner of the table, similar to how the plugin does it 🤷🏼

Yeah, I think this would be better. One thing I was thinking about is the current placement probably doesn't work so well for mobile. Will think about adjusting it on another PR.

**Pull request merged**